### ISSUE TITLE
Balm Crash due to Possibly Being Outdated

### ISSUE THREAD

**Issue by ENTPRESTIGIOUS** [ModdingForBlockheads/Forbic#16]
### Minecraft Version

1.18.x

### Mod Loader

Fabric

### Mod Loader Version

0.13.3-1.18.2

### Mod Version

1.18.1(Latest on Balm)

### Describe the Issue

i get a crash.
[crash-2022-03-04_16.05.20-client.txt](https://github.com/ModdingForBlockheads/Balm/files/8189400/crash-2022-03-04_16.05.20-client.txt)

Sorry for not uploading to gist or pastebin in response to "Logs"!

**Comment by ENTPRESTIGIOUS** [comment]
I was told by the Architectury team to request issue here.
[https://github.com/architectury/architectury-api/issues/215](https://github.com/architectury/architectury-api/issues/215)

**Comment by ghost** [comment]
They better fix this soon I'm not patient.

**Comment by smmmadden** [comment]
I'm also running into it but for Forge 1.18.2.  I checked the list of files and thought 2.4.3 would work, but it doesn't.
Version 2.4.3 for Minecraft 1.18.x  (guess this is really only for 1.18 and 1.18.1)
Updated forge build to 40.0.12 as well, but no good.  Here's the fatal error that prevents startup of (MP Server).
![image](https://user-images.githubusercontent.com/10309346/157360012-274ef257-45fa-4051-8ea8-d1cc392f566c.png)
Thanks!
Steve
